pkg/service/service.go

```diff
@@ -141,6 +144,8 @@ type Service struct {
 
 	lbmap         LBMap
 	lastUpdatedTs atomic.Value
+
+	l7lbSvcs map[string]uint16 // key: namespace/name, value: local proxy port
```

Comment:
Use `k8s.ServiceID` as the key.

Reply:
The service may not yet exist when this map is populated due to a CEC being applied. As such the serviceID would not be available at the time.